app/(playground)/p/[agentId]/beta-proto/left-menu/github-integration/server-actions.ts

```diff
@@ -0,0 +1,56 @@
+"use server";
+
+import { db, gitHubIntegrations } from "@/drizzle";
+import type {
+	GitHubNextAction,
+	GitHubTriggerEvent,
+} from "@/services/external/github/types";
+import type { GiselleNodeId } from "../../giselle-node/types";
+import type { GitHubIntegrationId } from "../../github-integration/types";
+import { generateId } from "../../github-integration/utils";
+import type { AgentId } from "../../types";
+
+interface SaveInput {
+	id: GitHubIntegrationId | undefined;
+	agentId: AgentId;
+	repositoryFullName: string;
+	callSign: string;
+	event: GitHubTriggerEvent;
+	nextAction: GitHubNextAction;
+	startNodeId: GiselleNodeId;
+	endNodeId: GiselleNodeId;
+}
+export async function save(input: SaveInput) {
+	const agent = await db.query.agents.findFirst({
+		where: (agents, { eq }) => eq(agents.id, input.agentId),
+	});
+	if (agent === undefined) {
+		throw new Error(`Agent with id ${input.agentId} not found`);
+	}
+	await db
+		.insert(gitHubIntegrations)
+		.values({
+			id: input.id ?? generateId(),
+			dbId: 123,
```

Comment:
question: Will this code be subject to changes in the future?

Reply:
Sorry! This is unnecessary code that was used during development.  I remove 634355ed850c4359ba11bdf875a7075a7d563fbb